conductor_api/src/conductor/base.rs

```diff
@@ -897,16 +911,17 @@ pub mod tests {
         let mut conductor = Conductor::from_config(config.clone());
         conductor.dna_loader = test_dna_loader();
         conductor.key_loader = test_key_loader();
-        conductor.load_config(None).unwrap();
+        conductor.load_config().unwrap();
         conductor
     }
 
     fn test_conductor_with_signals(signal_tx: SignalSender) -> Conductor {
         let config = load_configuration::<Configuration>(&test_toml()).unwrap();
         let mut conductor = Conductor::from_config(config.clone());
+        conductor.set_signal_sender(signal_tx);
         conductor.dna_loader = test_dna_loader();
         conductor.key_loader = test_key_loader();
-        conductor.load_config(Some(signal_tx)).unwrap();
+        conductor.load_config().unwrap();
```

Comment:
I kinda liked it having load_config take the Option<Sender>, so that way you have to declare whether you want it or not, instead of maybe forgetting it. What do you think?

Reply:
But most places don’t need to extract all signals from all instances. It’s only the waiter that does. And it’s completely safe to not set it when it’s not needed.